pkg/routeagent/controllers/route/route.go

```diff
@@ -357,7 +358,7 @@ func (r *Controller) processNextPod() bool {
 			return fmt.Errorf("error retrieving submariner-route-agent pod object %s: %v", name, err)
 		}
 
-		klog.V(4).Infof("In processNextPod, POD HostIP is %s", pod.Status.HostIP)
+		klog.V(log.DEBUG).Infof("In processNextPod, POD HostIP is %s", pod.Status.HostIP)
```

Comment:
I suggest making this Info so we can see each pod that was processed:

`klog.Infof("Processing submariner-route-agent pod %q with host IP %q, pod IP %q", name, pod.Status.HostIP, pod.Status.PodIP)`

Reply:
I think it's better to leave this one as debug (with the new formatting) since it will replicate in some cases the one in L376 which is the real action out of this processing.